modules/dashboard/php/NDB_Form_dashboard.class.inc

```diff
@@ -36,7 +36,7 @@ class NDB_Form_Dashboard extends NDB_Form
         $DB     = Database::singleton();
         $user   = User::singleton();
         $config = NDB_Config::singleton();
-        $site   = $user->getSiteName();
+        $site   = explode(';', $user->getSiteName());
```

Comment:
This doesn't look right. Why is getSiteName returning a semicolon delimited string?